features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -76,6 +78,16 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the dynamic optimisation for the Session Replay feature.
+         * This will enable the time limitation that Session Replay recording can use on the main thread, enabling it
+         * might reduce the session replay frames recorded by the SDK.
+         */
+        fun setDynamicOptimisation(dynamicOptimization: Boolean): Builder {
```

Comment:
we need to choose which spelling we use: `optimiSation` or `optimiZation`

Reply:
Yes, that's the question I always wanted to ask, do we prefer usually British English or American English in the code?

Reply:
We never had the explicit rule for this one. Maybe you can search across the codebase and see which one is used the most. Anyway, my point here is that it should be consistent naming for this declaration.

Reply:
So I choose `Optimization`, updated everywhere.